### ISSUE TITLE
Error running docker container

### ISSUE THREAD

**Issue by senniraf** [imitator-model-checker/imitator#152]
Hi,

I built a docker image on the `develop` branch. The build was successful. However, when I try to start the container I get the following error:
```plain
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/imitator/bin/imitator": stat /imitator/bin/imitator: no such file or directory: unknown.
```

**Comment by himito** [comment]
Hi @senniraf,

Thank you for pointing out this issue. 
It was fixed in 2fc6a3972d2da57fb8b7548309d29310e6c2981e

Please let me know if it works on your side.

Best,
Jaime

**Comment by senniraf** [comment]
Hi Jaime,

thanks for fixing the issue. It now works on my side.

Best,
Raffael
